presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7889,10 +7890,16 @@ public void testQuantifiedComparisonCornerCases(String query)
         );
         List<QueryTemplate.Parameter> quantifier = new QueryTemplate.Parameter("quantifier").of("ALL", "ANY");
         List<QueryTemplate.Parameter> lhs = new QueryTemplate.Parameter("lhs").of("1", "NULL");
-        List<QueryTemplate.Parameter> operator = new QueryTemplate.Parameter("operator").of("<", ">", "<=", ">=");
+        List<QueryTemplate.Parameter> operator = new QueryTemplate.Parameter("operator").of("=", "!=", "<", ">", "<=", ">=");
         QueryTemplate queryTemplate = new QueryTemplate("SELECT %lhs% %operator% %quantifier% (%subquery%)");
+        ImmutableList<String> excludedInPredicateQueries = ImmutableList.of(
+                "SELECT NULL != ALL (SELECT * FROM (SELECT 1 WHERE false) as empty_table)",
```

Comment:
On the other hand, the test cases are written in terms of parameter names and values, so it's asymmetric.